CHANGELOG.md

```diff
@@ -1,6 +1,132 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.12.0b1:
+	- Avoid executing self upgrade policy unnecessarily (ENT-3592)
+	- Add amazon_linux class to yum package module
+	- Introduce ability to set policy update bundle via augments (CFE-2687)
+	- Localize delete tidy in ha update policy (ENT-3659)
```

Comment:
I don't understand what this means.

Reply:
@nickanderson: ? Should I remove it or can you reword it in more understandable way? I believe "ha" means "HA", which is short for "High Availability"

Reply:
Yes, I know what HA means, and yes it should be uppercase.

> Localize delete tidy

Still doesn't make much sense to me.